cms/settings/base.py

```diff
@@ -850,3 +851,8 @@
 SLACK_NOTIFICATIONS_WEBHOOK_URL = env.get("SLACK_NOTIFICATIONS_WEBHOOK_URL")
 
 ONS_API_BASE_URL = env.get("ONS_API_BASE_URL")
+
+PUBLISHER_BACKEND = os.getenv("PUBLISHER_BACKEND", "kafka")
+KAFKA_SERVER = os.getenv("KAFKA_SERVER", "localhost:9092")
+KAFKA_TOPIC_CREATED_OR_UPDATED = os.getenv("KAFKA_TOPIC_CREATED_OR_UPDATED", "search-content-updated")
+KAFKA_TOPIC_DELETED = os.getenv("KAFKA_TOPIC_DELETED", "search-content-deleted")
```

Comment:
Let's avoid falling back on localhost paths in the base settings.
The rule of thumb for integrations like this:
1. are the key settings present in the env? In this case this would be `PUBLISHER_BACKEND` and `KAFKA_SERVER`. 
2. if the backend env var is "kafka" and we have the server env var, set up all the settings. If the backend env var is set to something else, use that

The bottom line is `localhost:9092` is not a good fallback for a prod env

Reply:
Yes, I was going to clean up all of this, as this is not the correct way at all. Currently, it's only serving me to test some things.